Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Dockerfile
```

Comment:
nitpick: I don't think we need this line here, since it's already obvious that this is a Dockerfile

Reply:
fixed